docs/Support-Tiers.md

```diff
@@ -0,0 +1,92 @@
+---
+last_review_date: "2025-03-28"
+---
+
+# Support Tiers
+
+Homebrew has three support tiers.
+These set expectations for how well Homebrew will run on a given configuration.
+
+## Tier 1
+
+A Tier 1 supported configuration is one in which:
+
+- you get the best experience using Homebrew
+- we will aim to fix bugs affecting this platform
+- we will not output warnings running on this platform
+- we have CI coverage for testing and building bottles for this platform
+- your support is best met through Homebrew's issue trackers
+
+### macOS
+
+For Tier 1 support, Homebrew on macOS must be all of:
```

Comment:
Something I was just reminded of: your Homebrew should also be installed on the storage inside your mac. External storage issues keep popping up, but I don't consider them Tier 1 supported